src/css/custom.scss

```diff
@@ -18,3 +18,8 @@
   --ifm-color-primary-lightest: rgb(146, 224, 208);
   --ifm-code-font-size: 95%;
 }
+
+.react-toggle {
```

Comment:
Please, instead it paste `colorMode: {
      defaultMode: 'light',
      disableSwitch: true,
    },` in themeConfig